src/components/Accordion/Accordion.tsx

```diff
@@ -60,9 +67,16 @@ const getElementsFromData = (data: AccordionDataProps[] = [], id: string) => {
           id={`${id}-panel-${index}`}
           key={index}
           dangerouslySetInnerHTML={{ __html: content.panel }}
+          maxHeight={panelMaxHeight}
+          overflow="auto"
```

Comment:
Overflow auto used to cause problems with the Y scrollbar in IE.  I used Browserstack to test with Edge and I did not see any issues, but I want to make sure we are all keeping an eye out for this issue.